compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Upgrade.hs

```diff
@@ -666,6 +666,8 @@ generateSrcFromLf env thisPkgId = noLoc mod
             LF.BTContractId -> (damlStdlibUnitId, LF.ModuleName ["DA", "Internal", "LF"])
             LF.BTOptional -> (damlStdlibUnitId, LF.ModuleName ["DA", "Internal", "Prelude"])
             LF.BTMap -> (damlStdlibUnitId, LF.ModuleName ["DA", "Internal", "LF"])
+            LF.BTGenMap -> (damlStdlibUnitId, LF.ModuleName ["DA", "Internal", "LF"])
+                -- GENMAP TODO: Verify module name once GenMap implemented in stdlib.
```

Comment:
Can we get a ticket for this and link it here, please? Same below.

Reply:
Sure! 👍 I'll add the map ticket #2256.